src/RestServer/RestController.Blockchain.cs

```diff
@@ -0,0 +1,287 @@
+#pragma warning disable IDE0051
+#pragma warning disable IDE0060
+
+using Microsoft.AspNetCore.Mvc;
+using Neo.IO;
+using Neo.IO.Json;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using Neo.SmartContract;
+using Neo.SmartContract.Native;
+using Neo.VM;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Neo.Plugins
+{
+    partial class RestController
+    {
+        /// <summary>
+        /// Get the lastest block hash of the blockchain 
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet("blocks/bestblockhash")]
+        public IActionResult GetBestBlockHash()
```

Comment:
Current instead of best?

Reply:
yeah, i'm with you. the route path currently is mainly based on the method name of the rpc api.